_layouts/home.html

```diff
@@ -6,10 +6,12 @@
     {%- if page.title -%}
       <h1 class="page-heading">{{ page.title }}</h1>
     {%- endif -%}
-  
+
+    <div class="gcse-search"></div>
```

Comment:
So the blog home page isn't generally intended as a landing page (although GA shows it's 16% of page views) :shrug: 

Would this be better placed in the header?

If so, the search results should probably be on a separate page, or a modal dialog

Reply:
Creating a separate page or a modal dialog should not be too difficult - do we have a preference?